frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.tsx

```diff
@@ -1,12 +1,43 @@
 import * as React from 'react';
+import * as _ from 'lodash';
+import { match as RMatch } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
+import { Grid, GridItem } from '@patternfly/react-core';
+import MonitoringDashboardGraph from './MonitoringDashboardGraph';
+import { queries } from './monitoringDashboardQueries';
+import MonitoringDasboardCountBlock from './MonitoringDashboardCountBlock';
+
+export interface MonitoringDashboardProps {
```

Comment:
We don't export types anymore. You can infer the type of a component through `React.ComponentProps<typeof MyComponent>`